### PR TITLE
target: adding support for CloudGate gateway.

### PR DESCRIPTION
**Pull request by DannyNe**
Pull request for Option CloudGate Helium hotspot light.

We are currently still undergoing DEWI certification for hotspot-light!
I would like to make this pull request already so that once the DEWI certification is done, we can be merged quickly.

Best Regards,

Danny

### CODE REVIEW

**Comment by madninja** [comment]
This would not be your final configuration right? 

1. You'd have a  settings.toml change to include a setting for your security chip? 
2. Will you be using this repo's GitHub releases system for upgrade or do you have your own? 

If this is not your actual product configuration it's not useful to have a PR here. Please contact DeWi for updated instructions

**Comment by DannyNe** [comment]
Hi,

1. You are correct, I merged all my changes and did one clean pull request, the settings.toml was in the .gitignore. sorry for that.
2. We are going to use both, our OTA server for upgrading the kernel and rootfs (security fixes) and github for the gateway-rs.

Best Regards,

Danny

File: README.md [hunk 1]
```diff
@@ -110,6 +111,7 @@ Note that platforms will be tested much faster if you join the development proce
 [MTCDT]: https://www.multitech.com/brands/multiconnect-conduit
 [resiot]: https://www.resiot.io/en/resiot-gateways/
 [cotx]: https://www.cotxnetworks.com/product/service_one
+[CloudGate]: https://www.option.com/
```
Comment (madninja) [inline README.md:114]: please supply a valid website

File: package/cloudgate/data/etc/init.d/helium_gateway [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+start() {
+	/rom/mnt/cust/usr/bin/monit -g helium restart all
+}
+
+stop() {
+	/rom/mnt/cust/usr/bin/monit -g helium stop all
+}
+
+restart() {
+	/rom/mnt/cust/usr/bin/monit restart helium_cg
```
Comment (madninja) [inline package/cloudgate/data/etc/init.d/helium_gateway:14]: why is this called `helium_cg`?

**Review by madninja — COMMENTED** [review]
Sorry for the delay here.. 

Could you address the questions in the PR review and rebase your branch on master?

**Comment by madninja** [comment]
@DannyNe bump.. if you're no longer interested we can close this PR

File: README.md (hunk 1 above)
Comment (DannyNe) [reply README.md:114]: this is the valid URL for our company.

File: package/cloudgate/data/etc/init.d/helium_gateway (hunk 2 above)
Comment (DannyNe) [reply package/cloudgate/data/etc/init.d/helium_gateway:14]: helium_cg is a process that waits until there is an internet connection and starts the gateway_rs.
restarting, in this case, sends a signal to this process "helium_cg" that will in his turn stop the gateway_rs and then restarts it (when there is an internet connection of course).

**Comment by DannyNe** [comment]
We are still interested in this pull request but still busy with DEWI approval.

**Pull request merged**